blosc/blosc2.h

```diff
@@ -1517,6 +1517,15 @@ BLOSC_EXPORT blosc2_frame* blosc2_frame_from_sframe(uint8_t *sframe, int64_t len
 BLOSC_EXPORT blosc2_schunk* blosc2_frame_to_schunk(blosc2_frame* frame, bool copy);
 
 
+/*********************************************************************
+  Directory utilities for needed for doing tests.
```

Comment:
Well, this can be useful for general users too, so I'd rewrite this as just "Directory utilities".

Reply:
Fixed in 4607ab374e512f47bc7a64117e9af8907c18fc81